### ISSUE TITLE
ose should be avalable for user defined varibility in Population simulations

### ISSUE THREAD

**Issue by tobiasK2001** [Open-Systems-Pharmacology/PK-Sim#567]
I think the dose should be avalable for user defined varibility as it makes perfect sense adding varibility there. The labelled dose is never the actual dose in e.g. a tablet. USP allows content of a drug to vary between 85 per cent and 115 per cent of the average content. For other applications this applies as well even more or less. So we might miss this variability in performing population simulation be default.

**Comment by Yuri05** [comment]
I guess it makes sense only for dose **per body weight**, not for the absolute dose?

**Comment by tobiasK2001** [comment]
No, also for the absolute dose and for every kind of application (like user defined intracelleular muscle as well)

**Comment by tobiasK2001** [comment]
No means here: yes I want to hav it for the absolute dose as well :-)

**Comment by Yuri05** [comment]
>No means here: yes I want to hav it for the absolute dose as well :-)

I guess only in case the dose was defined as absolute?
In case dose was defined per BW I would allow only variation of DosePerBW, not of the absolute dose

**Comment by msevestre** [comment]
I think it makes sense for any type of input (Dose in mg, mg/kg or mg/m2).  Provided that this does not create any technical issues

**Comment by tobiasK2001** [comment]
Yes!
